Update .Net Core and Emby 4

thoradia opened the pull request:
This updates .Net Core Runtime and Emby 4, as requested in the forum
Emby (legacy) does not run with current .Net Core Runtime anymore and is therefore dropped
.Net Core Runtime now only requires international components for unicode, other libraries are therefore dropped
The Emby 4 add-on is a bootstrap that installs the latest release when installed/updated
Maintenance should now be minimal (three packages of which one is a bootstrap)
I can backport this pull request

Jellyfin was also requested in the forum, but is rather disapointing when compared to Emby 4

CvH commented:
could you add a dummy package for legacy emby like https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/addons/service/tvheadend/package.mk ? Otherwise the old emby won't get cleaned up client side and it just stays there with an error.

For icu can't we just update https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/addons/addon-depends/chrome-depends/icu/package.mk ? Btw no problem if chrome is failing, need to get updated anyway.

CvH commented on packages/addons/tools/dotnet-runtime/package.mk:
```diff
@@ -1,54 +1,46 @@
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_NAME=".Net Core Runtime"
+PKG_ADDON_PROJECTS="any !RPi1 !Slice"
```
```suggestion
PKG_ADDON_PROJECTS="any !RPi1"
```
there is no slice anymore at master

thoradia commented:
Thank you for the review @CvH
I have created a broken add-on package for emby
I have merged the icu4c package into the icu package
I have moved the icu package into the root of addon-depends to hint that it is now shared by serveral add-ons
I have removed Slice
I hope that I did not break more than I fixed

CvH commented on packages/addons/addon-depends/icu/package.mk:
```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="icu"
+PKG_VERSION="67-1"
+PKG_SHA256="571d3723a8055d85e513b19b70b4f7f370da10df084c3bb598e0fb8333da0a16"
+PKG_LICENSE="https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE"
```
```suggestion
PKG_VERSION="67.1"
PKG_SHA256="8987b530d7954e9a1b8c041dcb8959221d8a794710302a26e800174d2d511ad2"
PKG_LICENSE="MIT"
```

thoradia replied:
I am not sure that [this](https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE) is a MIT license

CvH replied:
its MIT with additions that fall completely out of the standard, so tbh I have no real idea
`PKG_LICENSE="Custom"` might be the alternative

thoradia replied:
Done

CvH commented on packages/addons/addon-depends/icu/package.mk:
```diff
@@ -0,0 +1,24 @@
+PKG_SHA256="571d3723a8055d85e513b19b70b4f7f370da10df084c3bb598e0fb8333da0a16"
+PKG_LICENSE="https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE"
+PKG_SITE="http://www.icu-project.org"
+PKG_URL="https://github.com/unicode-org/icu/archive/release-${PKG_VERSION}.tar.gz"
```
```suggestion
PKG_URL="https://github.com/unicode-org/icu/archive/release-${PKG_VERSION//./-}.tar.gz"
```

thoradia replied:
Isn't it simpler to use PKG_VERSION="67-1"?

CvH replied:
yes it would, but the version is 67.1, just the download+tag is 67-1 for odd reasons

thoradia replied:
Done

CvH commented on packages/addons/addon-depends/icu/package.mk:
```diff
@@ -0,0 +1,24 @@
+configure_package() {
+  PKG_CONFIGURE_SCRIPT="${PKG_BUILD}/icu4c/source/configure"
+  PKG_CONFIGURE_OPTS_TARGET="--disable-extras \
+                             --disable-renaming \
```
```suggestion
                             --disable-layoutex \
                             --disable-renaming \
```

thoradia replied:
I will disable layout and layoutex according to [this](https://github.com/unicode-org/icu/blob/2e56c4264acac327b4a9064a6000c39b602ad833/docs/userguide/layoutengine/paragraph.md)

thoradia replied:
Done

CvH commented:
@thoradia tx, @awiouy should have a look and press merge if he is happy

awiouy commented:
Looks good to me
